### PR TITLE
Add github orkflow for bundle size reporting

### PR DESCRIPTION
**Pull request by robertgrzonka**
Closes: https://github.com/blitz-js/blitz/issues/747

### What are the changes and their implications?

Adds GitHub Action which automatically provides report of bundle size for each Pull Requests and creating/updating package.

### Checklist

- [ ] Tests added for changes
- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes

### CODE REVIEW

File: .github/workflows/compressed.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Compressed Size
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (flybayer) [inline .github/workflows/compressed.yml:6]: You need to add the `canary` branch here, because currently all PRs are to canary :)

Comment (robertgrzonka) [reply .github/workflows/compressed.yml:6]: Fair enough!

File: .github/workflows/compressed.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - canary
```
Comment (flybayer) [inline .github/workflows/compressed.yml:7]: Hmm, still not showing up. Try this, which is what we have for the other CI actions.

```suggestion
on:
  push:
    branches:
      - master
      - canary
  pull_request:
    branches:
      - master
      - canary
```

File: .github/workflows/compressed.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  push:
+    branches:
+      - master
+      - canary
+  pull_request:
+    branches:
+      - master
+      - canary
+  registry_package:
+    types: [published]
```
Comment (flybayer) [inline .github/workflows/compressed.yml:13]: Maybe this is preventing it from running? Because we aren't using the github package registry

Comment (robertgrzonka) [reply .github/workflows/compressed.yml:13]: That's exactly what I was going to type to you :) I am checking this on my private repo once again and I'm sure that in a moment we will have proper commit here :)

**Comment by robertgrzonka** [comment]
@flybayer we got this 🎉

**Comment by flybayer** [comment]
@all-contributors add @robertgrzonka for code

**Comment by flybayer** [comment]
Oops, wrong contrib type last time

@all-contributors add @robertgrzonka for infra

**Pull request merged**